modules/nf-core/kallisto/quant/main.nf

```diff
@@ -56,7 +56,11 @@ process KALLISTO_QUANT {
             ${reads} 2> >(tee -a ${prefix}.log.txt >&2)
 
     if [ -f $prefix/run_info.json ]; then
-        cp $prefix/run_info.json "${prefix}_run_info.json"
+        cp $prefix/run_info.json "${prefix}.run_info.json"
+    fi
+
+    if [ -f ${prefix}.log.txt ]; then
+        cp ${prefix}.log.txt ${prefix}/kallisto_quant.log
     fi
```

Comment:
We should update the nf-core/module with this change and then re-install here so we don't have to patch it.

Reply:
Actually this would be fixed by changing the stdout redirection above.

Reply:
We will need to keep that file outside of the directory because we need to pass it to MultiQC. So using this logic is the best way I think.

Reply:
OK, I'll make both write to the dir and copy out for consistency though.

Reply:
Think we're good though @drpatelh - if you could check that module PR to make sure you're happy, we can get it merged and updated here.